src/commands/run.js

```diff
@@ -5,13 +5,14 @@ const path = require('path');
 const execWithLog = require('../lib/exec');
 const { LOCAL_ENV_VARS, MAIN_FILE } = require('../lib/consts');
 const { ENV_VARS } = require('apify-shared/consts');
-const { getLocalUserInfo, purgeDefaultQueue, purgeDefaultKeyValueStore, purgeDefaultDataset } = require('../lib/utils');
+const { getLocalUserInfo, purgeDefaultQueue, purgeDefaultKeyValueStore, purgeDefaultDataset, getLocalConfigOrThrow } = require('../lib/utils');
 const { info } = require('../lib/outputs');
 
 class RunCommand extends ApifyCommand {
     async run() {
         const { flags } = this.parse(RunCommand);
         const { proxy, id: userId, token } = getLocalUserInfo();
+        const localConfig = getLocalConfigOrThrow();
         const apifyLocalEnvVars = LOCAL_ENV_VARS;
```

Comment:
You don't need to create variable thats equal to constant, just use constant and it will be clearer